bigtable/admin.go

```diff
@@ -275,6 +278,47 @@ func (ac *AdminClient) CreateColumnFamily(ctx context.Context, table, family str
 	return err
 }
 
+// TableConf contains all of the information necessary to create a table with column families.
+type UpdateTableConf struct {
+	TableID string
+	// Set to true to make the table protected against data loss
+	// i.e. deleting the table, the column families in the table and the instance containing the table would be prohibited
+	DeletionProtection *bool
+}
+
+// UpdateTable updates a table in the instance from the given configuration.
+// Only deletion protection can be updated at this period.
+func (ac *AdminClient) UpdateTable(ctx context.Context, conf *UpdateTableConf) (updated bool, err error) {
```

Comment:
Do we really need a bool? no error means the table is updated successfully, right?